manifests/02-namespace.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
     include.release.openshift.io/single-node-developer: "true"
     openshift.io/node-selector: ""
     workload.openshift.io/allowed: "management"
+    network.openshift.io/policy-group: "console"
```

Comment:
Well, I'm checking on mobile but AFAIS you add annotation but label is needed on our end